README.md

```diff
@@ -28,11 +59,46 @@ python boxsim.py <navigator> --ue
 python boxsim.py <navigator> --ue --save_images 'path/to/dataset'
 ~~~
 
-See the command line arguments listed in `boxsim.py` for more options.
+
+### Note about Command Line Arguments
+
+The above command specifies the required arguments needed to run the box simulation in Unreal Engine. But there are other optional specifications. Here are all the arguments you can specify:
+
+- \<navigator>: must choose between "wandering" and "perfect" and decides the navigator that will be used when running the simulation.
+- --anim_ext=\<extension>: (optional) extension for output format to help visualize run.
+- --max_actions=\<int>: (optional) Maximum number of allowed actions when running the simulation
+- --save_images=\<file path>: (optional) enables image capturing to specified path (may be relative or absolute path)
+- --ue: (optional) Note: Required if you want to connect to Unreal Engine
+- --py_port=\<port>: (optional) Python OSC server port.
+- --ue_port=\<port>: (optional) Unreal Engine OSC server port.
+- --image_ext=\<extension>: (optional) Sets file extensions for images to be saved as.
+- --resolution=\<string ResXxResY>: (optional) sets resolution of screen
+
+You can look at how these arguments are used in the boxsim.py file.
```

Comment:
We can probably delete this from here and just tell people to read the code. Otherwise we'll likely have these instructions fall out of sync with changes we make to boxsim.py.

Reply:
Sounds good.